apps/ledger-live-desktop/src/renderer/screens/settings/sections/General/ChangeDeviceLanguagePromptDrawer.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useCallback, useState } from "react";
+
+import { Flex, Drawer, Button, Divider } from "@ledgerhq/react-ui";
+import ChangeDeviceLanguageAction from "~/renderer/components/ChangeDeviceLanguageAction";
+import { useDispatch, useSelector } from "react-redux";
+import { Locale, localeIdToDeviceLanguage } from "~/config/languages";
+import { DeviceInfo } from "@ledgerhq/types-live";
+import { setLastSeenDevice } from "~/renderer/actions/settings";
+import { useTranslation } from "react-i18next";
+import { track } from "~/renderer/analytics/segment";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import { command } from "~/renderer/commands";
+import { getCurrentDevice } from "~/renderer/reducers/devices";
+import ChangeDeviceLanguagePrompt from "~/renderer/components/ChangeDeviceLanguagePrompt";
+import { DeviceModelId, getDeviceModel } from "@ledgerhq/devices";
+
```

Comment:
We should import `DeviceModelId` enum from `@ledgerhq/types-devices` now ;)

Reply:
fixed